dotcom-rendering/src/web/lib/renderElement.tsx

```diff
@@ -132,20 +131,13 @@ export const renderElement = ({
 	isAdFreeUser,
 	switches,
 	isSensitive,
-	abTests,
 }: Props): [boolean, JSX.Element] => {
 	const palette = decidePalette(format);
 
 	const isBlog =
 		format.design === ArticleDesign.LiveBlog ||
 		format.design === ArticleDesign.DeadBlog;
 
-	const isInteractivesIdleLoadingVariant =
-		abTests?.interactivesIdleLoadingVariant === 'variant';
-
-	const shouldIdleLoadInteractives =
-		!isAdFreeUser && isInteractivesIdleLoadingVariant;
```

Comment:
I decided to remove the special case where interactives are loaded lazily for ad-free users. This is because:

1) the improvement in CLS felt worth rolling out to a wider audience (even if they weren't affected by the [spacefinder issues](https://github.com/guardian/dotcom-rendering/pull/4942) this experiment set out to address)
2) it's one less special case to worry about

Happy to be challenged if anyone has strong views on this!